src/compliance/privacy/ccpa.md

```diff
@@ -0,0 +1,46 @@
+---
+group: compliance
+title: California Consumer Privacy Act
+---
+
+{: .bs-callout .bs-callout-info}
```

Comment:
```suggestion
{:.bs-callout-info}
```

`{: .bs-callout .bs-callout-info}` is the archaic way of creating a note. The preferred way is to omit the first `.bs-callout`.